proceedings.bib

```diff
@@ -134,6 +140,8 @@ @string{ ppsn_1998
 @string{ ppsn_2010      = "Parallel Problem Solving from Nature -- PPSN XI: 11th International Conference" }
 @string{ ppsn_2014      = "Parallel Problem Solving from Nature -- PPSN XIII: 13th International Conference" }
 
+@string{ robio_2010     = "2010 IEEE International Conference on Robotics and Biomimetics (ROBIO)" }
+@string{ robot_v1_2015  = "Robot 2015: Second Iberian Robotics Conference: Advances in Robotics, Volume 1" }
```

Comment:
I think the year should proceed the volume: robot_2015_v1

The volumes will belong to the proceedings of the conference of the same year and, therefore, the most important information is the year.